metadata-ingestion/src/datahub/sql_parsing/sql_parsing_aggregator.py

```diff
@@ -272,6 +274,8 @@ def compute_stats(self) -> None:
         self.num_temp_sessions = len(self._aggregator._temp_lineage_map)
         self.num_inferred_temp_schemas = len(self._aggregator._inferred_temp_schemas)
 
+        self.sql_parsing_cache_stats = sqlglot_lineage_cached.cache_info()._asdict()  # type: ignore
```

Comment:
there's also a cache wrapping `_parse_statement` and `try_format_query` - might as well get those stats too